README.md

```diff
@@ -36,19 +37,21 @@ module "aws-dashboards" {
   lightstep_project = "your-lightstep-project"
 }
 ```
-* Run `terraform init`
-* Add some code to define dashboards, streams, alerts, and more. [See documentation](https://registry.terraform.io/providers/lightstep/lightstep/latest/docs) for examples or use [pre-built Lightstep Terraform modules](https://registry.terraform.io/namespaces/lightstep).
-* After setting an environment variable with your API Key that matches the name in the provider configuration above, run `terraform plan` to preview changes.
+
+- Run `terraform init`
+- Add some code to define dashboards, streams, alerts, and more. [See documentation](https://registry.terraform.io/providers/lightstep/lightstep/latest/docs) for examples or use [pre-built Lightstep Terraform modules](https://registry.terraform.io/namespaces/lightstep).
+- After setting an environment variable with your API Key that matches the name in the provider configuration above, run `terraform plan` to preview changes.
 
 :warning: If you're creating many Lightstep resources at once, we recommend running the `apply` with the `parallelism` flag set to a low value to avoid API 500 errors:
 
 ```
    # Avoids 500 errors when creating many resources.
-   terraform apply -parallelism=1 
+   terraform apply -parallelism=1
 ```
+
 ## Development
 
-See `DEVELOPMENT.md`.
+See [`DEVELOPMENT.md`](DEVELOPMENT.md).
```

Comment:
Added the link here. All other changes to this file are cosmetic, Markdown-linting changes.